app/src/main/java/org/mozilla/tv/firefox/hint/HintBinder.kt

```diff
@@ -37,7 +37,6 @@ object HintBinder {
                                 .start()
                     }
                     .subscribe { shouldDisplay ->
-                        hintContainer.isVisible = true
```

Comment:
This isn't quite right: in the `animate` case, after we dispose once, the view will never be visible again because e set `isVisible` to false but this is the only line that sets `isVisible` to true.